bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterAdminGrpcClient.java

```diff
@@ -101,7 +101,6 @@ public void deleteCluster(DeleteClusterRequest request) {
   
   @Override
   public void close() throws Exception {
-    channel.close();
```

Comment:
The major functionality I'm adding here is channelOptions.addClientCloseHandler(..).  There were artifacts created in both BigtableChannels and BigtableOptionsFactory that needed closing; they both call addClientCloseHandler now.  AbstractBigtableConnection.close() does the closing of those artifacts